src/components/EventCard.tsx

```diff
@@ -15,16 +18,40 @@ export default function EventCard({
   starttime,
   endtime,
   location,
+  timezone,
 }: EventCardProps) {
+  const [numRespondents, setNumRespondents] = useState(0)
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    getNumRespondents(eventId).then((data) => {
```

Comment:
moving to bulk means this would have to get a set of events and make multiple calls to get the full data needed.